README.MD

```diff
@@ -67,7 +67,7 @@ geokit bbox2fc --bbox="0, 0, 10, 10" > output.geojson
 
 ### Buffer
 
-- Creates a buffer in LineString features for a given radius.
+- Creates features' buffer for a given radius.
```

Comment:
'Create a buffer of specified radius for all the features'

Reply:
yes, it is for all the features.